helm/ingress-azure/templates/crds.yaml

```diff
@@ -4,12 +4,26 @@
   {{ $.Files.Get $path }}
 ---
 {{- end }}
+{{- $watchNamespace := .Values.kubernetes.watchNamespace -}}
+{{- range .Values.appgw.prohibitedTargets -}}
 apiVersion: appgw.ingress.k8s.io/v1
 kind: AzureIngressProhibitedTarget
 metadata:
-  name: prohibit-all-targets
+  name: {{ .name }}
+  {{- if $watchNamespace }}
+  namespace: {{ $watchNamespace | splitList "," | first }}
+  {{- end }}
 spec:
+  {{- if .hostname }}
+  hostname: {{ .hostname }}
```

Comment:
I think it would be nice to install "/*" when user doesn't specify custom rules. What do you think ?

Reply:
I've added the existing `prohibit-all-targets` CRD if `appgw.prohibitedTargets` is missing or empty. This has the existing behaviour if you just specify `appgw.shared=true`